apps/music/style/music.css

```diff
@@ -266,12 +290,15 @@ body.picker-mode #title-player {
   overflow-y: auto;
   background: -moz-linear-gradient(center top, #303030 0%, #000000 100%) repeat scroll 0 0 transparent;
   z-index: 10;
-  transition: transform 300ms ease;
+  transition: transform 300ms ease, visibility 300ms;
+  visibility: hidden;
 }
 
 .view {
   width: 100%;
   height: auto;
+  visibility: hidden;
+  transition: visibility 300ms;
```

Comment:
Since it is a binary value (`visible`/`hidden`) it is not actually animated itself. `visibility` is designed to work correctly in conjunction with other transitions. For example, a transition from `visible` to `hidden` with 300ms duration will have it actually hidden at the end of 300ms. A transition from `hidden` to `visible` will work by having it becoming visible immediately. In this way, if the `visibility` property i combined with, for example, `opacity`, you will get the correct dissolve effect since the element will remain visible for the duration of the transition.

Did that answer your question? :)